backend.py

```diff
@@ -7,13 +7,50 @@
 import random
 from enum import Enum
 
+
 class Tile:
-    def __init__(self, direction, path):
+    def __init__(self, direction, path, type):
         self.direction = direction
         self.path = path
+        self.type = type
 
     def __repr__(self):
-        return "<Tile {} {}>".format(self.rotation, self.path)
+        return "<Tile {} {}>".format(self.direction, self.type)
+
+    def can_move_from(self, direction):
+        """
+        Verify movement from tile in specific direction.
+
+        Return a boolean.
+
+        True - There is not a wall in direction of the move.
+        False - There is a wall in direction of the move.
+        """
+        # The direction of the wall is the same as the direction in which
+        # robot wants to move from the tile.
+        if (self.type == "wall" and self.direction == direction):
+            move = False
+        else:
+            move = True
+        return move
```

Comment:
Do init doplň ještě "properties" za "type".a self.propesties=propesties

Reply:
Zatím je tam psaný `type`, protože pracuji s položkou `type` z JSONU. Až budu opravdu pracovat s `properties`, tak to přepíšu.